src/webgl/p5.RendererGL.js

```diff
@@ -1701,6 +1715,14 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
     return tex;
   }
 
+  // used in imageLight
+  // create a new texture from the img input
+  getBlurryTexture(input){
+    const blurrytex = new p5.Texture(this, input);
```

Comment:
nit: blurryTexture or inputBlurryTexture

Reply:
We actually refactored this function completely.